INFO - 2023-09-15 06:35:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-15 06:35:36 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'pc04project'@'localhost' (using password: YES) in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'pc04project', '0124896553', 'pc04project', 3306, '', 0)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#9 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-15 06:35:36 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Frontend->home()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-15 06:36:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-15 06:36:47 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'pc04project'@'localhost' (using password: YES) in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'pc04project', '0124896553', 'pc04project', 3306, '', 0)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#9 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-15 06:36:47 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Frontend->home()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-15 06:36:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-15 06:36:48 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'pc04project'@'localhost' (using password: YES) in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'pc04project', '0124896553', 'pc04project', 3306, '', 0)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#9 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-15 06:36:48 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Frontend->home()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-15 06:36:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-15 06:36:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'pc04project'@'localhost' (using password: YES) in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'pc04project', '0124896553', 'pc04project', 3306, '', 0)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#9 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-15 06:36:49 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Frontend->home()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-15 06:39:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-15 06:39:31 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'pc04project'@'localhost' (using password: YES) in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'pc04project', '0124896553', 'pc04project', 3306, '', 0)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#9 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-15 06:39:31 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Frontend->home()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-15 06:46:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-15 06:46:18 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'pc04project'@'localhost' (using password: YES) in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'pc04project', '0124896553', 'pc04project', 3306, '', 0)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#9 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-15 06:46:18 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Frontend->home()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-15 06:48:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-15 06:48:18 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'pc04project'@'localhost' (using password: YES) in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'pc04project', '0124896553', 'pc04project', 3306, '', 0)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#9 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-15 06:48:18 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Frontend->home()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-15 06:53:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 06:56:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 06:56:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 06:56:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 06:56:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 06:56:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 06:56:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 06:56:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 06:56:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 07:00:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 07:00:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 07:00:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 07:02:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 07:02:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 07:02:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 07:02:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 07:02:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 07:02:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 07:02:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 07:02:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 11:12:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-15 11:13:01 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'pc04project', '0124896553', 'pc04project', 3306, '', 0)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#9 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-15 11:13:01 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Frontend->home()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-15 11:13:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-15 11:13:10 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'pc04project'@'localhost' to database 'pc04project' in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'pc04project', '0124896553', 'pc04project', 3306, '', 0)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#9 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' to database '****' in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-15 11:13:10 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' to database '****'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Frontend->home()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-15 11:13:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-15 11:13:11 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'pc04project'@'localhost' to database 'pc04project' in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'pc04project', '0124896553', 'pc04project', 3306, '', 0)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#9 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' to database '****' in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-15 11:13:11 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' to database '****'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Frontend->home()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-15 11:13:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-15 11:13:11 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'pc04project'@'localhost' to database 'pc04project' in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'pc04project', '0124896553', 'pc04project', 3306, '', 0)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#9 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' to database '****' in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-15 11:13:11 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' to database '****'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Frontend->home()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-15 11:13:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-15 11:13:11 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'pc04project'@'localhost' to database 'pc04project' in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'pc04project', '0124896553', 'pc04project', 3306, '', 0)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#9 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' to database '****' in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-15 11:13:11 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' to database '****'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Frontend->home()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-15 11:13:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-15 11:13:11 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'pc04project'@'localhost' to database 'pc04project' in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'pc04project', '0124896553', 'pc04project', 3306, '', 0)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#9 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' to database '****' in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-15 11:13:11 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' to database '****'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Frontend->home()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-15 11:13:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-09-15 11:13:25 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'pc04project'@'localhost' to database 'pc04project' in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'pc04project', '0124896553', 'pc04project', 3306, '', 0)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#9 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' to database '****' in C:\xampp\htdocs\pc04project\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\pc04project\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\pc04project\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\pc04project\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\pc04project\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\pc04project\app\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(932): App\Controllers\Frontend->home()
#7 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
#8 C:\xampp\htdocs\pc04project\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\pc04project\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-15 11:13:25 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' to database '****'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Frontend.php(47): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Frontend->home()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Frontend))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2023-09-15 11:15:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-09-15 11:15:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
